src/store/reducer.ts

```diff
@@ -0,0 +1,32 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {getCameras, getCamera, getRequests} from './action';
+import { CamerasType, CameraType } from '../types/cameras';
+import { ReviewsType } from '../types/reviews';
+
+type InitalState = {
+  cameras: CamerasType;
+  camera: CameraType | null;
+  reviews: ReviewsType | null;
+}
+
+const initialState: InitalState = {
+  cameras: [],
+  camera: null,
+  reviews: null,
+};
+
+
+const reducer = createReducer(initialState, (builder) => {
+  builder
+    .addCase(getCameras, (state, action) => {
+      state.cameras = action.payload;
+    })
```

Comment:
как правило, пишут так
```suggestion
  builder
    .addCase(getCameras, (state, {payload}) => {
      state.cameras = payload;
    })
```